components/Navigation.js

```diff
@@ -0,0 +1,55 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const Navigation = () => {
+  const router = useRouter();
+
+  return (
+    <NavBar>
+      <NavItem $active={router.pathname === "/"}>
+        <Link href="/">I</Link>
+      </NavItem>
+    </NavBar>
+  );
+};
+
+export default Navigation;
+
+const NavBar = styled.nav`
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 90%;
```

Comment:
these 3 properties currently lead to some undesired behaviour, since your `nav` element slightly differs in its size depending on whether you are on the overview page or the details view. Most likely since the parent elements are different in size and % takes that as its reference.

I suggest you go with `vw` in this case, since it takes into account the width of the device and you don't need to use `transform` in this case either:

```suggestion
  left: 5vw;
  width: 90vw;
```